content-src/lib/getBestImage.js

```diff
@@ -0,0 +1,29 @@
+const IMG_HEIGHT = 226;
```

Comment:
I just moved this from `Spotlight.js` so it could be used at the selector level, instead of in the component